src/workflows-app/SubmissionDetails.test.ts

```diff
@@ -91,7 +146,46 @@ jest.mock('react-virtualized', () => {
   };
 });
 
-const captureEvent = jest.fn();
+const captureEvent: MockedFn<MetricsContract['captureEvent']> = jest.fn();
+
+interface MockCbasArgs {
+  getRuns?: MockedFn<CbasAjaxContract['runs']['get']>;
+  getRunsSets?: MockedFn<CbasAjaxContract['runSets']['get']>;
+  getMethods?: MockedFn<CbasAjaxContract['methods']['getById']>;
+}
+const mockCbas = (fns: MockCbasArgs) => {
```

Comment:
added mockCbas helper fn that is used by mockCbasWith fn below.  MockCBasWith combined with breaking apart Ajax mocking (no more parent Ajax() to worry about), and some related cleanup, greatly reduces some heavy code repetition I saw in the //Arrange sections of all the tests in this file.